src/modules/Bookmark/controller.ts

```diff
@@ -42,4 +42,39 @@ const createBookmark = async (req: Request, res: Response) => {
   }
 };
 
-export default { createBookmark };
+const getBookmarkedArticle = async (
+  req: Request,
+  res: Response,
+): Promise<void> => {
+  const { bookmarkId } = req.params;
+
+  try {
+    const id = parseInt(bookmarkId, 10);
+    if (isNaN(id)) {
+      res.status(400).send({ message: 'Invalid bookmark ID.' });
+      return;
+    }
```

Comment:
this would be checked by the validation middleware beforehand, just wondering if it is needed again here? @ahzia